docker-compose.yml

```diff
@@ -28,3 +33,54 @@ services:
       RABBITMQ_DEFAULT_VHOST: /libretime
       RABBITMQ_DEFAULT_USER: libretime
       RABBITMQ_DEFAULT_PASS: libretime
+
+  playout:
+    build: ./playout
+    volumes:
+      - ./docker/config.yml:/etc/libretime/config.yml:ro
+
+  liquidsoap:
+    build: ./playout
+    command: /usr/local/bin/libretime-liquidsoap
+    ports:
+      - "1234:1234"
+    volumes:
+      - ./docker/config.yml:/etc/libretime/config.yml:ro
+
+  analyzer:
+    build: ./analyzer
+    volumes:
+      - ./docker/config.yml:/etc/libretime/config.yml:ro
```

Comment:
Missing storage mount.

Reply:
It downloads via REST, so doesn't need the storage mount

Reply:
I don't think it's downloaded through the HTTP API, a path is send through rabbitmq and that's all. I think only playout downloads through the HTTP API endpoint.